packages/slider/slider.ts

```diff
@@ -42,6 +42,13 @@ class WarpSlider extends FormControlMixin(LitElement) {
   @property({ type: Boolean, reflect: true })
   disabled = false;
 
+  @property({ type: Boolean, attribute: 'allow-values-outside-range' })
+  allowValuesOutsideRange = false;
+
+  // TODO Implement error texts
```

Comment:
Can we convert this TODO to an issue with a bit of an explanation of the goal/problem we want to solve?